cfn-update-script/cfn-template-update.py

```diff
@@ -71,6 +73,18 @@ class SyncPair:
     tgt_id:str
     tgt_reg:str
 
+def check_version():
+    # Checking Python version:
+    expect_major = 3
+    expect_minor = 12
+    expect_rev = 8
+    current_version = str(version_info[0])+"."+str(version_info[1])+"."+str(version_info[2])
+    print("INFO: Script developed and tested with Python " + str(expect_major) + "." + str(expect_minor) + "." + str(expect_rev))
+    if version_info[0] < (expect_major) or version_info[1] < (expect_minor):
```

Comment:
I think this logic is incorrect with new major versions of Python with smaller minors, like 4.0 (because 0 < 12), but maybe that's not a practical problem.  Another way of doing this is letting Python compare tuples directly:

    if (version_info[0], version_info[1]) < (expect_major, expect_minor):
        do_things()

It'll do the right thing with numbers (compare the first elements in the tuple, and then only if they're equal, move to the next, etc.).